tests/cases/fourslash/refactorConvertStringOrTemplateLiteral_TemplateString2.ts

```diff
@@ -0,0 +1,15 @@
+/// <reference path='fourslash.ts' />
+
+////const foo = 1;
+////const bar = /*start*/"a" + `${foo}`/*end*/;

```

Comment:
It looks like the refactor will delete strings following the substitution part, e.g.
```ts
"a" + `${foo}` + "b"
```